Elevation level
uint8 shelf
string direction
---
bool success
